resources/lib/pigskin.py

```diff
@@ -228,6 +228,31 @@ def get_weeks_games(self, season, season_type, week):
             raise
 
         return sorted(games, key=lambda x: x['gameDateTimeUtc'])
+        
+    def get_team_games(self, season, team=None):
+        try:
+            url = self.config['modules']['ROUTES_DATA_PROVIDERS']['teams']
+            teams = self.make_request(url, 'get')
+            if team is None:
+                return teams
+            else:
+                #Looking for the Teamname
```

Comment:
Sorry, I should have been more specific: comments usually have a space following the "#"

As I said, it's a nitpick and something I can fix when merging.